src/istio/mixerclient/quota_cache.cc

```diff
@@ -218,9 +227,12 @@ void QuotaCache::SetResponse(const Attributes& attributes,
 
   utils::HashType signature;
   if (!referenced.Signature(attributes, quota_name, &signature)) {
-    GOOGLE_LOG(ERROR) << "Quota response referenced mismatchs with request";
-    GOOGLE_LOG(ERROR) << "Request attributes: " << attributes.DebugString();
-    GOOGLE_LOG(ERROR) << "Referenced attributes: " << referenced.DebugString();
+    if (MIXER_WARN_ENABLED) {
+      MIXER_WARN(
+          "Quota response referenced mismatchs with request. Request "
```

Comment:
same as above; only I note here that we use WARN instead of ERROR.  Is that intentional?

Reply:
I think  we should pick one threshold for both this one and the one above.  I'm tempted to say both of these are WARN - they shouldn't have happened, someone needs to look at them, but only the transaction is borked, the proxy itself can carry on happily.  Thoughts?

Reply:
WARN sounds good to me.